Made changes to the AssignmentController and the created Junit tests for

Kyoshinono opened the pull request:
the controller.

Kyoshinono committed:
Made changes to the AssignmentController and the created Junit tests for
the controller.

Kyoshinono commented:
I have reviewed the code and have not seen anything to change. The testing for the delete function may need to be reworked, not sure how to test that the repository performed a delete.

Kyoshinono committed:
Added Junit for Assignment.

yyakhnis commented:
Remove any import statement that is underlined in yellow (i.e. not used), particularly in the AssignmentController class.

fromJsonString method, line 274, JunitTestAssignment is never used, so it could be scrapped.

line 86, AssigmentController, variable is never used, so it should be scrapped.

Kyoshinono committed:
Removed all unused import statements in assignmentController.
Kept fromJsonString method in case it is needed for future testing.
removed the email variable on line 86 of AssignmentController.
